.github/workflows/romcal-ci.yml

```diff
@@ -6,10 +6,37 @@ on:
   pull_request:
     types: [opened, reopened, synchronize]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
```

Comment:
Does this cancel previous jobs in progress or if there are other jobs in progress, it cancels this one? 🤔

Reply:
it cancels any job that isn't the latest